charts/redpanda/Chart.yaml

```diff
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 5.0.9
+version: 6.0.0
```

Comment:
What are we breaking from `5.0.9`? We're just adding a feature, right? `5.1.0` maybe?

Reply:
I can try after https://github.com/redpanda-data/helm-charts/pull/649

Reply:
https://github.com/redpanda-data/helm-charts/actions/runs/5801748835/job/15726685420?pr=646#step:18:170

`--reuse-values` is used and it prevents from render correct Connectors Deployment.

Reply:
you have set this to 5.0.10, why did you not decide to go to 5.1.0?

Reply:
I can go to 5.1.0 no problem. It will be in set in the next push.